src/recipe.h

```diff
@@ -24,7 +24,7 @@
 #include <libxml/tree.h>
 
 // XXX make this configurable
```

Comment:
Possible way to go?

Reply:
Still need a default or everyone will break.